src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -527,18 +622,15 @@ public static void Register(Type type, Type serializerType)
                 {
                     Register(
                         type,
-                        obj =>
-                        {
+                        obj => {
```

Comment:
several of these are superfluous but also incorrect according to our guidelines. It's better to keep the diff small, but especially if the changes conflict with our guidelines (as sometimes fixing some styling errors is nice if they don't take the focus of the PR)